package/MDAnalysis/core/universe.py

```diff
@@ -385,33 +407,39 @@ def empty(cls, n_atoms, n_residues=1, n_segments=1,
         residue_segindex: array like, optional
           mapping of residues to segments
         trajectory: bool, optional
-          if True, attaches a :class:`MDAnalysis.coordinates.memory.MemoryReader`
-          allowing coordinates to be set and written.  Default is False
+          if ``True``, attaches a
+          :class:`MDAnalysis.coordinates.memory.MemoryReader` allowing
+          coordinates to be set and written.  Default is ``False``
```

Comment:
```suggestion
          coordinates to be set and written.
```

Let's start removing default values from docstrings?